packages/block-library/src/navigation/edit/navigation-menu-selector.js

```diff
@@ -235,6 +237,14 @@ function NavigationMenuSelector( {
 							>
 								{ __( 'Create new menu' ) }
 							</MenuItem>
+							<MenuItem
+								disabled={ ! canManageMenus }
+								href={ addQueryArgs( 'edit.php', {
+									post_type: 'wp_navigation',
+								} ) }
+							>
+								{ __( 'Manage menus' ) }
+							</MenuItem>
```

Comment:
This UI is effectively representing a standard `<select>` component. I don't know for certain but my instinct is that it's not going to be accessible for what is effectively a standard link to be inside the select.

That said, when it's inside the experiment that's a different matter. In that case the link will be inside a dropdown from the panel's 3 dots menu which I think will be fine.